### PR TITLE
feat(322): Add links to footer

### PR DESCRIPTION
**Pull request by tkyi**
Need some advice:
- not sure about wording/keywords for links
- not sure what logo to display on the bottom and where it should link to when clicked (currently points to https://developer.yahoo.com)
- do we still want the links to the `theme` and `photo` source? is that the best place to put them?

screen shot 2016-12-14 at 4 26 11 pm

Related to https://github.com/screwdriver-cd/screwdriver/issues/322

### CODE REVIEW

**Comment by stjohnjohnson** [comment]
Ehh, I'm not 100% sold on the style.  What examples are you pulling from?

**Comment by tkyi** [comment]
Most sites have links listed under topic headers
http://concourse.ci/
https://travis-ci.org/
http://www.wercker.com/

The CSS is just default. I don't really know what proper font sizing would be

File: _includes/css/screwdriver.css [hunk 1]
```diff
@@ -12,6 +12,31 @@
     font-size: 17px;
 }
 
+footer .row {
+    font-size: 15px;
```
Comment (petey) [inline _includes/css/screwdriver.css]: I think this is too big. Try 12px.

File: _includes/css/screwdriver.css [hunk 2]
```diff
@@ -12,6 +12,31 @@
...
+    text-align: left;
+}
+
+footer .row h4 {
+    padding-left: 40px;
```
Comment (petey) [inline _includes/css/screwdriver.css]: Header Font also too big. Try `font-size: 14px`

File: _includes/css/screwdriver.css [hunk 3]
```diff
@@ -12,6 +12,31 @@
...
+    list-style: none;
+    text-align: left;
+}
+
+footer .row .citation {
+    font-size: 11px;
+    padding-left: 0px;
+}
+
+footer .yahoo-logo {
```
Comment (petey) [inline _includes/css/screwdriver.css]: I'd get rid of the logo entirely.

File: _includes/css/screwdriver.css [hunk 4]
```diff
@@ -12,6 +12,31 @@
...
+footer .row {
+    font-size: 15px;
+    text-align: left;
+}
+
+footer .row h4 {
+    padding-left: 40px;
+}
+
+footer .row .quicklinks {
```
Comment (petey) [inline _includes/css/screwdriver.css]: I think the line height is kinda big. Try `line-height: 25px;`

File: _includes/css/screwdriver.css [hunk 5]
```diff
@@ -12,6 +12,31 @@
...
+    list-style: none;
+    text-align: left;
+}
+
+footer .row .citation {
+    font-size: 11px;
```
Comment (petey) [inline _includes/css/screwdriver.css]: not-needed, it can be 12px like the rest.

File: _includes/footer.html [hunk 6]
```diff
@@ -1,26 +1,56 @@
...
                         </li>
-                        {% endfor %}
                     </ul>
                 </div>
-                <div class="col-md-4">
-                    <ul class="list-inline quicklinks">
+            <div>
+                <div class="col-md-3">
+                    <ul class="quicklinks">
+                    <li>
```
Comment (petey) [inline _includes/footer.html]: Lose the logo.

File: _includes/footer.html [hunk 7]
```diff
@@ -1,26 +1,52 @@
...
+                        <li><a href="http://stackoverflow.com/questions/tagged/screwdriver-cd">Stack Overflow</a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="col-md-3">
+                </div>
+                <div class="col-md-3">
+                    <ul class="quicklinks">
+                    <li><span class="copyright">Copyright Yahoo Inc. {{ 'now' | date: "%Y" }}</span>
+                    </li>
```
Comment (minz1027) [inline _includes/footer.html]: Can we move this `</li>` to line 37 to make them in the same line?

File: _includes/footer.html [hunk 8]
```diff
@@ -1,26 +1,52 @@
...
                         </li>
                         <li>
-                            <a href="https://flic.kr/p/hmZTt">Photo by Don DeBold</a>
+                            <a href="https://github.com/y7kim/agency-jekyll-theme">Theme</a> by Rick K.
                         </li>
-                    </ul>
+                        <li>
+                            <a href="https://flic.kr/p/hmZTt">Photo</a> by Don DeBold
+                        </li>
+                    </li></ul>
```
Comment (minz1027) [inline _includes/footer.html]: The order of `</li></ul>` doesn't look right..

**Comment by d2lam** [comment]
Hmmm is it a lot of work to put the `Copyright Yahoo Inc.` with on the bottom in _1 single line_? Right now it looks like there are 4 columns with different spacing

**Comment by tkyi** [comment]
@d2lam Yeah the uneven columns is intentional. I was using https://travis-ci.org/ as reference. I'm not sure how the spacing would work if I moved the `Copyright Yahoo` part to another line. The 3 columns will look kind of awkwardly spaced.

screen shot 2016-12-15 at 11 01 45 am

Do you prefer this?

File: _includes/footer.html (hunk 7 above)
Comment (tkyi) [inline _includes/footer.html]: Left it on another line but fixed indentation

**Pull request merged**